src/app/shared/auth/auth-guard.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@angular/core';
+import { Router, CanActivate } from '@angular/router';
+import { AuthService } from './auth.service';@Injectable()
+export class AuthGuardService implements CanActivate {
+  constructor(public auth: AuthService, public router: Router) {}  canActivate(): boolean {
+    if (!this.auth.isAuthenticated()) {
+      this.router.navigate(['']);
+      this.auth.setLoginStatus(false);
+      return false;
+    }
+    return true;
```

Comment:
```
if (this.auth.isAuthenticated()) {
     return true
    }
    this.router.navigate(['login']);
    
   return false;
```

Reply:
I don't have login page. Regarding task's purpose I chose creating popup window for login form